src/App.jsx

```diff
@@ -1,21 +1,38 @@
 import './App.css'
 import TodoList from "./TodoList.jsx";
 import AddTodoForm from "./AddTodoForm.jsx";
-import {useState} from "react";
+import {useEffect, useState} from "react";
 
+function useSemiPersistentState() {
```

Comment:
I think these changes were from last week's assignment? If so, I wouldn't expect to see them in this MR. Did you merge your work from last week into the main branch?